src/lib/krb5/krb/get_in_tkt.c

```diff
@@ -1604,7 +1604,7 @@ warn_des3(krb5_context context, krb5_init_creds_context ctx,
         return;
 
     banner = _("Warning: encryption type des3-cbc-sha1 used for "
-               "authentication is weak and will be deprecated");
+               "authentication is weak and will be desupported");
```

Comment:
I think this change is good in general but "desupported" is a bit unclear.  "will be nsupported in the future" is clearer, but long; maybe "will be removed"?

Reply:
I'll change it to "will be disabled" since that is the next step.